app/views/layouts/local-template.html.erb

```diff
@@ -61,6 +61,7 @@
   </head>
 
   <body class="<%= content_for?(:body_classes) ? raw("#{yield(:body_classes)}") : "#{config_item(:phase).to_s.downcase} #{config_item(:product_type).to_s.downcase}" %>">
+    <%= render :partial => 'shared/tag_manager' %>
```

Comment:
A code style improvement: `<%= render 'shared/tag_manager' %>` - the hash is not necessary.